### PR TITLE
Cleanup usage of Dict

### PR DESCRIPTION
**Pull request by gmile**
Since Dict is being deprecated, clean up the usage of it

### CODE REVIEW

File: lib/phoenix/controller.ex [hunk 1]
```diff
@@ -734,7 +734,6 @@ defmodule Phoenix.Controller do
 
   defp to_map(assigns) when is_map(assigns), do: assigns
   defp to_map(assigns) when is_list(assigns), do: :maps.from_list(assigns)
-  defp to_map(assigns), do: Dict.merge(%{}, assigns)
```
Comment (gmile) [inline lib/phoenix/controller.ex]: I'm very uncertain about this change, but the tests passed.

File: lib/phoenix/view.ex [hunk 2]
```diff
@@ -250,7 +250,7 @@ defmodule Phoenix.View do
 
   """
   def render_existing(module, template, assigns \\ []) do
-    render(module, template, Dict.put(assigns, :render_existing, {module, template}))
+    render(module, template, Keyword.put(assigns, :render_existing, {module, template}))
```
Comment (josevalim) [inline lib/phoenix/view.ex]: I think there is no guarantee it is a keyword on this one. I would suggesting using:

    put_in assigns[:render_existing], {module, template}

**Comment by josevalim** [comment]
Looks great, I have added just one tiny comment.

@chrismccord we want to include this in v1.2.

**Comment by gmile** [comment]
@josevalim thanks, I just pushed a fix to that one

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**